src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -100,7 +100,7 @@ public abstract partial class TestKitBase
             CancellationToken cancellationToken = default)
         {
             return FishForMessageAsync(isMessage, allMessages, max, hint, cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.